packages/json-schema/test/utils.ts

```diff
@@ -10,21 +11,21 @@ export async function getHostForCadlFile(contents: string, decorators?: Record<s
     libraries: [JsonSchemaTestLibrary],
   });
   if (decorators) {
-    await host.addJsFile("dec.js", decorators);
+    host.addJsFile("dec.js", decorators);
```

Comment:
Does this not do blocking i/o?

Reply:
nah, it was showing an ide "warning" that this await aws unecessary. In the test host this is just a in memory thing